packages/vm/lib/runBlock.ts

```diff
@@ -84,49 +83,6 @@ export interface RunBlockResult {
   receiptRoot: Buffer
 }
 
-/**
```

Comment:
Technically removing these is a breaking change. To avoid it you can re-export what's now in runTx.

Reply:
thanks, done in https://github.com/ethereumjs/ethereumjs-monorepo/pull/1185/commits/a6c622e486246765953425c06ef9a1b427bf536f